scripts/upgrade

```diff
@@ -49,6 +49,41 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
+#=================================================
+# ENSURE DOWNWARD COMPATIBILITY
+#=================================================
+ynh_script_progression --message="Ensuring downward compatibility..."
+
+if ynh_compare_current_package_version --comparison le --version 2023.02.22~ynh1
```

Comment:
```suggestion
if ynh_compare_current_package_version --comparison lt --version 2023.02.22~ynh1
```

Maybe you mean lower than (strict, not lower-or-equal), otherwise upgrade from that very same version (2023.02.22) ends up going through this block

Reply:
Thanks Alex for your comment. 
I indeed want it to go through that block